ruby/gps_pub.rb

```diff
@@ -0,0 +1,7 @@
+require 'ably'
+
+lat, long = 10.1233, 12.22
```

Comment:
No need to show the values.  In fact, this makes the code sample more questionable given that's hard coded.

Reply:
Fixed